src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Resources.resx

```diff
@@ -385,7 +385,7 @@
   <data name="PhaseMustBeInitialized" xml:space="preserve">
     <value>The phase must be initialized.</value>
   </data>
-  <data name="RazorProject_PathMustStartWithForwardSlash" xml:space="preserve">
+  <data name="RazorProjectFileSystem_PathMustStartWithForwardSlash" xml:space="preserve">
```

Comment:
I would expect to see this update a bunch of culture specific files too? Do we just not have translations for these strings?

Reply:
I would expect that too, but I don't see any in the repo: https://github.com/search?q=repo%3Adotnet%2Frazor%20RazorProject_PathMustStartWithForwardSlash&type=code.

Reply:
Where would compiler translated strings be picked up from? It looks like the XLIFF stuff is turned off for the compiler:

https://github.com/dotnet/razor/blob/a53884aecd188bffd294f1cccae08a2f25beec78/src/Compiler/Directory.Build.props#L22

Reply:
FWIW, I don't see any resources for Microsoft.CodeAnalysis.Razor.Compiler.dll when I look at the installed .NET SDK directory tree. Are we actually producing localized resources for the Razor compiler? cc @jaredpar

Reply:
AFAIK the Razor compiler has never been localized. eg, https://github.com/dotnet/razor/issues/7560, https://github.com/dotnet/razor/issues/7177

Reply:
Believe that is correct, we're not producing localized resources here.

Reply:
Appreciate the sanity check! In that case, I think I'm off the hook for that. :smile: